src/pages/studyView/StudyViewUtils.spec.ts

```diff
@@ -1806,22 +1789,31 @@ describe('StudyViewUtils', () => {
 
             assert.isTrue(
                 isOccupied(
-                    [['1', '1', ''], ['2', '2', '']],
+                    [
+                        ['1', '1', ''],
+                        ['2', '2', ''],
+                    ],
                     { x: 0, y: 0 },
                     { w: 1, h: 1 }
                 )
             );
             assert.isTrue(
                 isOccupied(
-                    [['1', '1', ''], ['2', '2', '']],
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki